pages/project-version/update/submit/index.js

```diff
@@ -14,6 +14,19 @@ module.exports = settings => {
     next();
   });
 
+  app.use((req, res, next) => {
+    const params = {
+      projectId: req.project.id,
+      establishmentId: req.establishmentId
+    };
+    req.user.can('project.endorse', params)
+      .then(canEndorse => {
+        res.locals.static.canEndorse = canEndorse;
```

Comment:
Is this not already accessible from `allowedActions`?

Reply:
ooh probably